### PR TITLE
Fixed the installation issue of istio in prows

### PR DESCRIPTION
**Pull request by houshengbo**

### CODE REVIEW

**Comment by houshengbo** [comment]
@l-qing Cloud you rebase this pr?

**Comment by l-qing** [comment]
> @l-qing Cloud you rebase this pr?

My PR has been merged. 😁 https://github.com/knative/operator/pull/1391

File: test/e2e-tests.sh [hunk 1]
```diff
@@ -29,9 +29,6 @@
 source $(dirname $0)/e2e-common.sh
 
 function knative_setup() {
-  if is_ingress_class istio; then
-    install_istio || fail_test "Istio installation failed"
-  fi
```
Comment (dsimansk) [inline test/e2e-tests.sh:34]: Is it intentional to remove the installation of Istio completely?

Comment (houshengbo) [reply test/e2e-tests.sh:34]: right, it has been run twice.

**Pull request merged**